test/integration.spec.js

```diff
@@ -33,14 +33,14 @@ async function executeAndGetLogs(fixture, globalPlugins) {
   }
 }
 
-it('integration : success', async() => {
+it('integration : success', async () => {
```

Comment:
please dont make changes in the style that make the pr less readable

Reply:
reverted!